.github/workflows/docs-ci.yaml

```diff
@@ -39,6 +39,8 @@ on:
         type: string
         default: html
 
+permissions:
+  pull-requests: write
```

Comment:
PR description says `continue-on-error-comment` is added but I don't see it here, just this new permissions block which seems related. I assume this is pending [feedback on Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1725402169663899)?

Reply:
Missed a couple bits when I ripped `continue-on-error-comment` back out -- fixed.